docs/adrs/0001-architecture.md

```diff
@@ -0,0 +1,16 @@
+## Architecture
+
+You can use any of the [supported operating systems](https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners), and the compatible `architecture` can be selected using `architecture`. Values are `x86`, `x64`, `arm`, `arm64`, `amd64` (not all of the architectures are available on all platforms).
```

Comment:
"...and the compatible `architecture` can be selected using `architecture`"
I would switch this with something like:
". Compatible architecture can be selected by specifying a value for 'architecture' input variable."